File: 백준/Bronze/31428. 엘리스 트랙 매칭/엘리스 트랙 매칭.cs
int N = int.Parse(Console.ReadLine());
string[] friends = Console.ReadLine().Split();
string HelloBit = Console.ReadLine();

int count = 0;
foreach (string friend in friends)
{
    if (friend == HelloBit)
    {
        count++;
    }
}

Console.WriteLine(count);

File: 백준/Bronze/31428. 엘리스 트랙 매칭/엘리스 트랙 매칭.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int N = sc.nextInt();
        sc.nextLine(); // 개행 문자 처리
        String[] friends = sc.nextLine().split(" ");
        String HelloBit = sc.nextLine();

        int count = 0;
        for (String friend : friends) {
            if (friend.equals(HelloBit)) {
                count++;
            }
        }

        System.out.println(count);
    }
}
